### PR TITLE
Make address unique identifier, restrict stat min window size to 300x300

### LINKED ISSUE #201: Uniquely identify properties by address

### PR DESCRIPTION
**Pull request by trxe**
closes #201 

as described

### CODE REVIEW

File: src/test/java/seedu/address/testutil/TypicalProperties.java [hunk 1]
```diff
@@ -34,10 +34,10 @@
 public class TypicalProperties {
 
     public static final Property P_ALICE = new PropertyBuilder()
-            .withName("Jurong West Ave 6")
+            .withName("Jurong West St 62")
             .withPrice(MICROSCOPIC_PRICE)
             .withSeller("Alice Pauline")
-            .withAddress("123, Jurong West Ave 6, #08-111")
+            .withAddress("123, Jurong West St 62, #08-111")
```
Comment (lizchow) [inline src/test/java/seedu/address/testutil/TypicalProperties.java:40]: Just a quick question: whats the point of this change?

File: src/test/java/seedu/address/model/property/PropertyTest.java [hunk 2]
```diff
@@ -33,22 +33,21 @@ public void isSameProperty() {
...
-        // name differs in case, all other attributes same -> returns true
-        Property editedBob = new PropertyBuilder(P_BOB).withName(VALID_NAME_BOB.toLowerCase()).build();
+        // address differs in case, all other attributes same -> returns true
+        Property editedBob = new PropertyBuilder(P_BOB).withAddress(VALID_ADDRESS_BOB.toLowerCase()).build();
         assertTrue(P_BOB.isSameProperty(editedBob));
 
-        // name has trailing spaces, all other attributes same -> returns false
-        String nameWithTrailingSpaces = VALID_NAME_BOB + " ";
-        editedBob = new PropertyBuilder(P_BOB).withName(nameWithTrailingSpaces).build();
+        // address has trailing spaces, all other attributes same -> returns false
```
Comment (lizchow) [inline src/test/java/seedu/address/model/property/PropertyTest.java:48]: I think this returns true?

```suggestion
        // address has trailing spaces, all other attributes same -> returns true
```

**Review by lizchow — APPROVED** [review]
lgtm! Remember to fix the user guide also especially the valid property name section https://ay2122s1-cs2103t-w11-4.github.io/tp/UserGuide.html#valid-propertiesbuyers.

File: src/test/java/seedu/address/testutil/TypicalProperties.java (hunk 1 above)
Comment (trxe) [reply src/test/java/seedu/address/testutil/TypicalProperties.java:40]: Another default property uses this address as well, so I changed it to fix the tests

**Pull request merged**